py/desitarget/streams/io.py

```diff
@@ -0,0 +1,318 @@
+"""
+desitarget.streams.io
+=====================
+
+Reading/writing data for the MWS Stellar Stream programs.
+"""
+from time import time
+
+import os
+import fitsio
+import numpy as np
+import healpy as hp
+import astropy.coordinates as acoo
+import astropy.units as auni
+
+from desitarget import io
+from desitarget.geomask import pixarea2nside, add_hp_neighbors, sweep_files_touch_hp
+from desitarget.gaiamatch import match_gaia_to_primary
+from desitarget.targets import resolve
+from desitarget.streams.utilities import betw
+from desiutil import depend
+
+# ADM set up the DESI default logger.
+from desiutil.log import get_logger
+log = get_logger()
+
+# ADM start the clock.
+start = time()
```

Comment:
Similar to earlier comment: is this intended to start a timer at package import time, vs. when a function is run?

Reply:
Agreed. I moved the start inside the relevant function.